### DISCUSSION TITLE
Cannot set Times New Roman font with mpdf 8.2.4

### DISCUSSION THREAD

**Discussion by arax35** [mpdf/mpdf#2066] [Q&A] [answered]
I am new with mpdf. I installed mpdf 8.2.4. No matter what I do, I can't get Times New Roman correctly with greek chars.
I have the following code:

```php
<?php
$html = file_get_contents('pdf.html');

require_once __DIR__ . '/vendor/autoload.php';
$defaultConfig = (new \Mpdf\Config\ConfigVariables())->getDefaults();
$fontDirs = $defaultConfig['fontDir'];

$defaultFontConfig = (new \Mpdf\Config\FontVariables())->getDefaults();
$fontData = $defaultFontConfig['fontdata'];

$mpdf = new \Mpdf\Mpdf([
	'mode' => 'utf-8',
	'format' => 'A4',
	'default_font_size' => 16,
	'fontDir' => array_merge($fontDirs, [
        __DIR__ . '/vendor/mpdf/mpdf/roman',
    ]),
    'fontdata' => $fontData + [ // lowercase letters only in font key
        'roman' => [
           'R' => "timesnewroman.ttf",
	   'B' => "timesnewroman-bold.ttf",
           'I' => "timesnewroman-italic.ttf",
           'BI' => "timesnewroman-bold-italic.ttf",
        ]
    ],
    'default_font' => 'roman'
]);
$mpdf->SetDefaultBodyCSS('background', "url('images/border.png')");
$mpdf->SetDefaultBodyCSS('background-image-resize', 6);
$mpdf->SetDefaultBodyCSS('margin-top', '10px');
$mpdf->WriteHTML($html);
$mpdf->Output('filename.pdf','D');
?>
```

I have made a folder named roman in vendor/mpd/mpdf and put in there ttf files for Times New Roman with the exact names with R, B, I and BI. In my html file I have set in header:

```html
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
```

I have also set:

```css
body {
   font-family: roman;
}
```

but tried also:

```css
p {
   font-family: roman;
}
```

with no success. I keep getting small squares instead of greek letters. Numbers and special chars are shown correctly though. What am I doing wrong?

**Comment by finwe** [comment, marked as answer]
The setting seems OK. Does the used font really contain greek characters? Can you share the font files and an example HTML code?

**Comment by arax35** [comment]
It was the ttf files!! I opened those files and could see greek chars with no problem. I replaced them this the ones for Times New Roman in my c:\windows\fonts folder that my Win 10 pc uses and the problem disappeared!! Thank you very much!
